MediaBrowser.WebDashboard/Api/PackageCreator.cs

```diff
@@ -1,161 +0,0 @@
-    {
-        private readonly string _basePath;
-        private readonly IResourceFileManager _resourceFileManager;
-
-        public PackageCreator(string basePath, IResourceFileManager resourceFileManager)
-        {
-            _basePath = basePath;
-            _resourceFileManager = resourceFileManager;
-        }
-
-        public async Task<Stream> GetResource(
-            string virtualPath,
-            string mode,
-            string localizationCulture,
-            string appVersion)
-        {
-            var resourcePath = _resourceFileManager.GetResourcePath(_basePath, virtualPath);
-            Stream resourceStream = File.OpenRead(resourcePath);
-
-            if (resourceStream != null && IsCoreHtml(virtualPath))
-            {
-                bool isMainIndexPage = string.Equals(virtualPath, "index.html", StringComparison.OrdinalIgnoreCase);
-                resourceStream = await ModifyHtml(isMainIndexPage, resourceStream, mode, appVersion, localizationCulture).ConfigureAwait(false);
-            }
-
-            return resourceStream;
-        }
-
-        public static bool IsCoreHtml(string path)
-        {
-            if (path.IndexOf(".template.html", StringComparison.OrdinalIgnoreCase) != -1)
-            {
-                return false;
-            }
-
-            return string.Equals(Path.GetExtension(path), ".html", StringComparison.OrdinalIgnoreCase);
-        }
-
-        /// <summary>
-        /// Modifies the source HTML stream by adding common meta tags, css and js.
-        /// </summary>
-        /// <param name="isMainIndexPage">True if the stream contains content for the main index page.</param>
-        /// <param name="sourceStream">The stream whose content should be modified.</param>
-        /// <param name="mode">The client mode ('cordova', 'android', etc).</param>
-        /// <param name="appVersion">The application version.</param>
-        /// <param name="localizationCulture">The localization culture.</param>
-        /// <returns>
-        /// A task that represents the async operation to read and modify the input stream.
-        /// The task result contains a stream containing the modified HTML content.
-        /// </returns>
-        public static async Task<Stream> ModifyHtml(
-            bool isMainIndexPage,
-            Stream sourceStream,
-            string mode,
-            string appVersion,
-            string localizationCulture)
-        {
-            string html;
-            using (var reader = new StreamReader(sourceStream, Encoding.UTF8))
-            {
-                html = await reader.ReadToEndAsync().ConfigureAwait(false);
-            }
-
-            if (isMainIndexPage && !string.IsNullOrWhiteSpace(localizationCulture))
-            {
-                var lang = localizationCulture.Split('-')[0];
-
-                html = html.Replace("<html", "<html data-culture=\"" + localizationCulture + "\" lang=\"" + lang + "\"", StringComparison.Ordinal);
```

Comment:
who's going to do this now?

Reply:
I should probably update the CI to pull the standalone build in this pull request. The language should be handled entirely based on what a user has for their browser settings, which I honestly prefer to this method.